src/contractor/contractor.cpp

```diff
@@ -412,8 +461,7 @@ std::size_t Contractor::LoadEdgeExpandedGraph(
         std::ofstream datasource_stream(datasource_indexes_filename, std::ios::binary);
         if (!datasource_stream)
         {
-            throw util::exception("Failed to open " + datasource_indexes_filename +
-                                  " for writing");
+            throw util::exception("Failed to open " + datasource_indexes_filename + " for writing");
         }
```

Comment:
@MoKob performance - it's done here because (a) all the we need is available, and (b) to avoid looping over the same data again.  I did it this way to minimize the additional time added to `osrm-contract`